# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/vnc/xfree86-4-4.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./xc.patch.orig	2004-09-13 06:45:46.143510936 +0200
+++ ./xc.patch	2004-09-13 06:46:03.017945632 +0200
@@ -8,11 +8,11 @@
 + XCOMM
 + XCOMM X VNC server
 + XCOMM
-+ MFBDIR = mfb
-+ CFB8DIR = cfb
-+ CFB16DIR = cfb16
-+ CFB24DIR = cfb24
-+ CFB32DIR = cfb32
++ MFBSUBDIR = mfb
++ CFB8SUBDIR = cfb
++ CFB16SUBDIR = cfb16
++ CFB24SUBDIR = cfb24
++ CFB32SUBDIR = cfb32
 + XVNCDDXDIR = vnc/Xvnc
 + XVNCDIRS = $(STDDIRS) $(MFBDIR) \
 + 	    $(CFB8DIR) $(CFB16DIR) $(CFB24DIR) $(CFB32DIR) \
@@ -116,60 +116,6 @@
 !     nlwMiddle >>= 3; \
       while (h--) { \
   	srcpix = psrc[srcy]; \
-*** xc/programs/Xserver/cfb/cfbglblt8.c.orig	Fri Dec 14 19:59:23 2001
---- xc/programs/Xserver/cfb/cfbglblt8.c	Tue Aug 12 10:05:57 2003
-***************
-*** 284,288 ****
-      register glyphPointer   glyphBits;
-      register int	xoff;
-! #if defined(USE_LEFT_BITS) || (!defined(STIPPLE) && !defined(USE_STIPPLE_CODE))
-      register CfbBits	*dst;
-  #endif
---- 284,288 ----
-      register glyphPointer   glyphBits;
-      register int	xoff;
-! #if defined(USE_LEFTBITS) || (!defined(STIPPLE) && !defined(USE_STIPPLE_CODE))
-      register CfbBits	*dst;
-  #endif
-***************
-*** 292,296 ****
-      CfbBits		*dstLine;
-      CfbBits		*pdstBase;
-! #ifdef USE_LEFT_BITS
-      CARD32		*cTmp;
-  #endif
---- 292,296 ----
-      CfbBits		*dstLine;
-      CfbBits		*pdstBase;
-! #ifdef USE_LEFTBITS
-      CARD32		*cTmp;
-  #endif
-***************
-*** 399,403 ****
-  	    	} while (--hTmp);
-  	    	break;
-! #else /* !USE_LEFT_BITS */
-  	    	{
-  		    int h;
---- 399,403 ----
-  	    	} while (--hTmp);
-  	    	break;
-! #else /* !USE_LEFTBITS */
-  	    	{
-  		    int h;
-***************
-*** 412,416 ****
-  	    	glyphBits = clips;
-  	    	/* fall through */
-! #endif /* USE_LEFT_BITS */
-  	    case rgnIN:
-  #ifdef STIPPLE
---- 412,416 ----
-  	    	glyphBits = clips;
-  	    	/* fall through */
-! #endif /* USE_LEFTBITS */
-  	    case rgnIN:
-  #ifdef STIPPLE
 *** xc/programs/Xserver/cfb/cfbcppl.c.orig	Fri Dec 14 19:59:22 2001
 --- xc/programs/Xserver/cfb/cfbcppl.c	Sun Apr 18 12:53:36 2004
 ***************
--- ./xc/config/cf/vnc.def.orig	2004-09-13 06:50:23.325849986 +0200
+++ ./xc/config/cf/vnc.def	2004-09-13 06:51:52.378509287 +0200
@@ -6,7 +6,8 @@
 #define BuildNls NO
 #define BuildXIE NO
 #define BuildGlxExt NO
-#define XnestServer NO
+#define XnestServer YES
+#define XF86Server NO
 #define XprtServer NO
 
 #ifdef SunArchitecture

